src/fprime/fpp/cli.py

```diff
@@ -9,6 +9,7 @@
 from typing import Callable, Dict, List, Tuple
 
 from fprime.fpp.common import FppUtility
+from fprime.fbuild.builder import Build
```

Comment:
Is this unused here, or was something missed?

Reply:
It's used for type hints and was doing the quoted version
```
(build: "Build")
```
which was not resolving correctly in VSCode